docs/dev-process/pull-request-process.md

```diff
@@ -0,0 +1,51 @@
+# Pull Request Process
+
+Requirements for a pull request to be accepted into PowerShell
+* Writing tests
+* Writing documentation
+
+## Pull Request Workflow
+
+1. The PR *author* creates a pull request from a fork.
+1. The *author* ensures that their pull request passes the [CI system][ci-system] build.
+  - If the build fails, a [Repository Maintainer][repository-maintainer] adds the `Review - waiting on author` label to the pull request. 
+  The *author* can then continue to update the pull request until the build passes.
+3. If the *author* knows whom should participate in the review, they should add them otherwise they can add the recommended *reviewers*.
+1. Once the build passes, if there is not sufficient review, the *maintainer* adds the `Review - needed` label.
+1. An [Area Expert][area-expert] should also review the pull request.
+  - If the *author* does not meet the *reviewer*'s standards, the *reviewer* makes comments. A *maintainer* then removes the `Review - needed` label and adds the `Review - waiting on author` label. The *author* must address the comments and repeat from step 2.
+  - If the *author* meets the *reviewer*'s standards, the *reviewer* approves the PR. A maintainer then removes the `need review` label.
+6. Once the code review is completed, a *maintainer* merges the pull request after one business day to allow for additional critical feedback.
+
+### Roles and Responsibilities
+
+1. The PR *author* is responsible for moving the PR forward.
+This includes addressing feedback within a timely period and indicating feedback has been addressed by adding a comment and mentioning the specific *reviewers*.
+1. *Reviewers* are responsible for ensuring the code addresses the issue being fixed, does not create new issues (functional, performance, or reliability), and implements proper design.
+*Reviewers* should use the `Review changes` drop down to indicate they are done with their review.
+  - `Request changes` if you believe the PR merge should be blocked if your feedback is not addressed,
+  - `Approve` if you believe your feedback has been addressed or the code is fine as-is
+  - `Comment` if you are making suggestions that the *author* does not have to accept.
+  Early in the review, it is acceptable to provide feedback on coding style based on the published [Coding Guidelines](coding-guidelines), however, after
+  the *author* has already pushed commits to address feedback, it is generally _not_ acceptable to focus on style issues as it disrupts the PR process.
+  Late feedback can be submitted as a new issue or new pull request from the *reviewer*.
```

Comment:
I think this depends on whether the late feedback is critical.
If maintainers can enforce the 1-day-action policy, then no comment is a late comment. Do you mean to refer to 'late comment' as comments made after approval(s)?

Reply:
Yes, I'll clarify that.

Reply:
I agree with @daxian-dbw. What is "style issues"? If the commits contains bad code, bad patterns and broken formattings we should request obligatory changes in any time. Any policy that allow pass bad PR is bad policy.
Currently we fast merge with ignoring minor issues only "blocking" and "critical" PRs. Also I think we can fast merge if author ask us (busy, vacation...) but before that, we have to open the tracking Issue and determine (PR Assignee) exactly who will do fix it as soon as possible. In fact, we are doing so.